conduct/code-of-conduct.md

```diff
@@ -31,34 +35,43 @@ harassment purposes.
+If you are being harassed, notice that someone else is being harassed, or have any other concerns relating to harassment, please contact a member of event staff immediately. Event staff can be identified by t-shirts/staff badges onsite; and an organizer can be found at the event registration counter at any time. You can contact the `<RESPONSE TEAM>` via `<CONTACT INFO>`.
 
-This code of conduct applies to R-Ladies spaces, but if you are being harassed by a member/guest/participant of/at R-Ladies outside our spaces, we still want to know about it. We will take all good-faith reports of harassment by R-Ladies members, especially the Global Leadership Team, seriously. This includes harassment outside our spaces and harassment that took place at any point in time. The Global Leadership Team reserves the right to exclude people from R-Ladies based on their past behavior, including behavior outside R-Ladies spaces and behavior towards people who are not in R-ladies.
+While this code of conduct applies to `<EVENT>` spaces, but if you are being harassed by a member/guest/participant of/at `<EVENT>` outside our spaces, we still want to know about it. We will take all good-faith reports of harassment by `<EVENT>` members, especially the organizing team, seriously. This includes harassment outside our spaces and harassment that took place at any point in time. The organizing team reserves the right to exclude people from `<EVENT>` based on their past behavior, including behavior outside `<EVENT>` spaces and behavior towards people who are not part of `<EVENT>`.
 
 In order to protect volunteers from abuse and burnout, we reserve the right to reject any report we believe to have been made in bad faith. Reports intended to silence legitimate criticism may be deleted without response.
 
-We will respect confidentiality requests for the purpose of protecting victims of abuse. At our discretion, we may publicly name a person about whom we've received harassment complaints, or privately warn third parties about them, if we believe that doing so will increase the safety of R-Ladies members or the general public. We will not name harassment victims without their affirmative consent.
+We will respect confidentiality requests for the purpose of protecting victims of abuse. At our discretion, we may publicly name a person about whom we've received harassment complaints, or privately warn third parties about them, if we believe that doing so will increase the safety of `<EVENT>` members or the general public. We will not name harassment victims without their affirmative consent.
+
+Conference staff will also provide support to victims, including, but not limited to:
 
+- Providing an escort
+- Contacting hotel/venue security or local law enforcement
+- Briefing key event staff to for response/victim assistance
+- And otherwise assisting those experiencing harassment to ensure that they feel safe for the duration of the conference.
 
```

Comment:
I'm not sure where to mention this on the document, but it might be worthwhile mentioning something about appropriate debriefing for the victim, and also those involved in reporting/assisting from conference staff. This helps ensure that those who help report or witness the event are safe and supported. The debriefing could also help provide feedback on how these situations are handled, so that they can be improved.